src/main/java/com/Elessar/app/ClientConsole.java

```diff
@@ -170,6 +161,7 @@ public static void main(String[] args) {
                                     logger.info("User {} log off successfully", currUser);
                                     currUser = "";
                                     msgQueue = null;
+                                    msgQueue.close();
```

Comment:
hehe, you would get a null pointer exception here lol

Reply:
oops...